Package.swift

```diff
@@ -505,6 +505,11 @@ let package = Package(
                 "TuistCoreTesting",
                 "TuistSigningTesting",
                 "TuistGraphTesting",
+            ],
+            exclude: [
+                "Target.Debug.p12",
+                "SignApp.debug.mobileprovision",
+                "Target.Debug.cer",
```

Comment:
Why do we need them now?

Reply:
this feels like it should not be relevant as we have dropped the signing features.

Reply:
SPM was throwing warnings because these files are part of the source directory but aren't sources & aren't excluded in the manifest.

Reply:
> this feels like it should not be relevant as we have dropped the signing features.

Should we remove the files entirely?

Reply:
Let's remove them, then 🚀